lua/packer/display.lua

```diff
@@ -708,4 +727,8 @@ end
 --- Async prompt_user
 display.ask_user = a.wrap(prompt_user)
 
+display.open_log = function()
+  if display.status.disp then display.status.disp:open_log() end
```

Comment:
Perhaps we should separate displaying the log from having an active display? e.g. by opening a new display if there isn't already one, and additionally creating a command to open the log?

Reply:
> We could also implement a command like `PackerLog` to open the log file.

Do you mean something like this? That would be a good idea.

> by opening a new display if there isn't already one

I don't quite understand what you mean by this. Are you saying we should open a packer display if there isn't one? What if we just provide a command which calls `log.open()`?

Reply:
Yes, sorry - I saw your `PackerLog` comment right after I finished my review.

> I don't quite understand what you mean by this. Are you saying we should open a packer display if there isn't one? What if we just provide a command which calls log.open()?

Unless I misread the code, it currently only opens the log display if there is already a `packer` results window open, correct? What I was proposing was that the command should *always* display the logs whether or not there's already a `packer` window.

Reply:
> Unless I misread the code, it currently only opens the log display if there is already a packer results window open, correct? 

Yes, only in the packer window.

> What I was proposing was that the command should always display the logs whether or not there's already a packer window.

Right, understood.